src/library/scala/Unit.scala

```diff
@@ -51,6 +51,7 @@ object Unit extends AnyValCompanion {
    *  @throws     ClassCastException  if the argument is not a scala.runtime.BoxedUnit
    *  @return     the Unit value ()
    */
+  @annotation.nowarn("cat=w-flag-value-discard")
```

Comment:
Another rock-solid dispensation from the other warning: you never worry about discarding boxed unit, which is just unit but in a nice box.

When a package arrives from Amazon, my problem is that I hate to discard the box.